Bottom: ca2388b9fec3713cd1c0ee00adb0ce3ac2baf5a0
Top:    1b4e7e5134e771cdaec380329c9b6fc42c20b6c3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 20:14:29 +0100

Refresh of draw-c-let-the-guis-manage-the

---

diff --git a/src/draw.c b/src/draw.c
index 53757f4..d13b151 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -115,15 +115,11 @@ AddPart (void *b)
 void
 Draw (void)
 {
-  HideCrosshair ();
-
   /* clear and create event if not drawing to a pixmap
    */
   if (Block.X1 <= Block.X2 && Block.Y1 <= Block.Y2)
     gui->invalidate_lr (Block.X1, Block.X2, Block.Y1, Block.Y2);
 
-  RestoreCrosshair ();
-
   /* shrink the update block */
   Block.X1 = Block.Y1 =  MAXINT;
   Block.X2 = Block.Y2 = -MAXINT;
